addons/account/tests/test_account_journal_dashboard.py

```diff
@@ -27,6 +27,81 @@ def assertDashboardPurchaseSaleData(self, journal, number_draft, sum_draft, numb
         dashboard_data = journal._get_journal_dashboard_data_batched()[journal.id]
         self.assertDictEqual({**dashboard_data, **expected_values}, dashboard_data)
 
+    def init_test_data(self, journal):
```

Comment:
You could make the name more explicit for what it does, e.g.
```suggestion
    def _create_test_vendor_bills(self, journal):
```